custom/wasm/client/rest/new_tx.go

```diff
@@ -0,0 +1,86 @@
+package rest
```

Comment:
It's a copy. I think we cannot leave it out. The `RegisterRESTRoutes` function imports functions of all the three files `tx.go,` `new_tx.go` and `query.go`. But you cannot import them from upstream, because they are not exported.

Reply:
let's me try out rest

Reply:
which code logic line in simulation has the custom taxes handling?

Reply:
rest still functions here

<img width="1512" alt="Screenshot 2023-05-31 at 12 33 10" src="https://github.com/classic-terra/core/assets/23058164/d19a959a-9ff4-4c35-ad95-8e675e761a3c">

Reply:
This is why. B struct embeds A struct which inherits all A functions. When B defines a new func with same signature as that of A, it will override. That is why custom AppModuleBasic{} already has all functions from wasm AppModuleBasic{}

<img width="1512" alt="Screenshot 2023-05-31 at 12 38 16" src="https://github.com/classic-terra/core/assets/23058164/0b32d7d1-46bb-414b-a7f8-d8152ccf5e11">